src/Forms/GridField/GridFieldGroupDeleteAction.php

```diff
@@ -41,6 +41,15 @@ public function getColumnContent($gridField, $record, $columnName)
         return null;
     }
 
+    public function getGroup($gridField, $record, $columnName)
```

Comment:
Couldn't find a unit test for this class or any reference to `GridFieldGroupDeleteAction` under `tests/php/Forms/GridField`.

Looks like there's a bit of conditional logic in this class that should be tested.

Reply:
Have added a unit test.